tests/vmi_multiqueue_test.go

```diff
@@ -143,7 +143,7 @@ var _ = Describe("[Serial]MultiQueue", func() {
 			Expect(xml.Unmarshal([]byte(domain), domSpec)).To(Succeed())
 
 			for i, iface := range domSpec.Devices.Interfaces {
-				expectedIfaceName := fmt.Sprintf("tap%d-nic", i)
```

Comment:
Please remove commit https://github.com/kubevirt/kubevirt/pull/4552/commits/c1e305e4d9757fdd23506b4f48906561a96cf0a6 instead of fixing in a new one.